transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/auth/AuthHandler.kt

```diff
@@ -0,0 +1,17 @@
+package com.genesys.cloud.messenger.transport.auth
+
+import com.genesys.cloud.messenger.transport.core.Empty
+import com.genesys.cloud.messenger.transport.core.Result
+
+internal const val NO_JWT = "ZW1wdHlfand0"
```

Comment:
I was wondering, what are these values for `NO_JWT` and `NO_REFRESH_TOKEN`?

Reply:
These values represents the state of jwt and refreshToken when they are either unknown, unset or when there were a critical failure related to auth flow. Potentially I could use null, but I wanted to avoid null checks.

Reply:
I was concerned the string values had a special meaning, but after we talked I see that they're just base64 encodings approximating the variable names 😀